.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: Curl | Bash Test
+on:
+  push:
+    paths:
+      - install.sh.in
+      - .github/workflows/ci.yml
+    # branches: [ main ]
+  schedule:
+    - cron: "0 0 * * 0"   # weekly
+jobs:
+  shellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - run: sudo apt install -y shellcheck
+      - run: shellcheck -S error install.sh.in
+
+  curl-bash:
+    needs: shellcheck
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        container:
+          - "ubuntu:latest"
+          - "ubuntu:24.10"
+          - "ubuntu:24.04"
+          - "ubuntu:22.04"
+          - "ubuntu:20.04"
+          - "debian:latest"
+          - "debian:12"
+          - "debian:11"
+          - "debian:10"
+          - "kalilinux/kali-rolling:latest"
+
+    container:
+      image: ${{ matrix.container }}
+
+    steps:
+      - name: apt
+        run: apt-get -y update && apt-get -y install curl gnupg2 gnupg gpg || true
+      - name: install ${{ matrix.container }}
+        run: curl -s https://install.zerotier.com | bash
```

Comment:
Thinking this should test the `install.sh.in` in the repo, and not the currently hosted version.  This tests what is currently hosted at install.zerotier.com, and not any changes that are in the commit/PR